site/content/en/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand.md

```diff
@@ -34,6 +34,15 @@ To install the visibility API, run the following command
 kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/{{< param "version" >}}/visibility-api.yaml
 ```
 
+### Install API Priority and Fairness:
+
+To install [API Priority and Fairness](https://kubernetes.io/docs/concepts/cluster-administration/flow-control/) for visibility API:
+
+{{< tabpane lang="shell" persist=disabled >}}
+{{< tab header="Kubernetes 1.29.x and newer" >}} kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/{{< param "version" >}}/apf.yaml {{< /tab >}}
```

Comment:
Can we just reference the configs in the configuration rather than entire installation manifests? WDYT @tenzen-y ?

Reply:
sorry, I missed this is just p&f configuration rather than whole manifests, in that case this part lgtm